### DISCUSSION TITLE
Highlight Color for Branch Search Results

### DISCUSSION THREAD

**Discussion by Pumbaazz** [gitextensions/gitextensions#11804] [Q&A] [answered]
I'm using the remote search branch feature and found out that when the result's highlight is very difficult to see. Do we have any setting or something that able to change this color?

The app version is:
- Git Extensions 4.1.0.16698
- Build 1fe52a137acef1fbd2b653dea263b44ad5844177
- Git 2.40.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.21
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

![image](https://github.com/gitextensions/gitextensions/assets/37480725/7e634ef3-13fd-4115-b0b4-a18f689ab503)

**Comment by RussKie** [comment]
I don't think there was.

The [current release is 4.2.1](https://github.com/gitextensions/gitextensions#downloads), which is newer, and I do believe there were certain changes affecting colours in that version (we those kind of changes pretty much in every version). I also welcome you to try out the [latest dev version](https://github.com/gitextensions/gitextensions#downloads), because again it may have some new visual changes over the v4.2.

**Reply by pmiossec** [reply, marked as answer]
> I don't think there was.

You're right, it is not possible (even if it was once possible when themes were working). It use a system color `SystemColors.Info` (system colors used for tooltips)

https://github.com/gitextensions/gitextensions/blob/master/src/app/GitUI/LeftPanel/RepoObjectsTree.cs#L366

**Reply by Pumbaazz** [reply]
Great, for some reason I'm unable to install the latest version. So do you think this should be added into a new feature in the latest version?

**Reply by RussKie** [reply]
So, you haven't verified whether it is a problem in the upcoming version but insist that it should be added as a feature...

**Reply by Pumbaazz** [reply]
Not really, I read the provided code from @pmiossec and know that it's not configurable for now and it's just a suggestion for a new feature, not insist on anything 😁
